server/src/main/java/org/apache/cassandra/sidecar/routes/cdc/Service.java

```diff
@@ -26,9 +29,27 @@ public enum Service
     CDC("cdc");
 
     public final String serviceName;
+    private static final Map<String, Service> LOOKUP = new HashMap<>();
+
+    static
+    {
+        for (Service service : Service.values())
+        {
+            LOOKUP.put(service.serviceName, service);
+        }
+    }
 
     Service(final String serviceName)
     {
         this.serviceName = serviceName;
     }
+
+    public static Service withName(String serviceName)
+    {
+        if (LOOKUP.containsKey(serviceName))
+        {
+            return LOOKUP.get(serviceName);
+        }
+        throw new RuntimeException("Invalid service name " + serviceName);
+    }
```

Comment:
This method is unnecessary. You can remove it. But if you want to provide detailed error context. Here is my suggestion
```suggestion
    public static Service withName(String serviceName)
    {
        try
        {
            return Service.valueOf(serviceName);
        }
        catch (Exception cause)
        {
            throw  new RuntimeException("Invalid service name " + serviceName, cause);
        }
    }
```

Reply:
Enum.valueOf() is case sensitive, so when service name is lowercase valueOf() throws exception